rootfs/api/models/app.py

```diff
@@ -129,7 +132,65 @@ def delete(self, *args, **kwargs):
         self._clean_app_logs()
         return super(App, self).delete(*args, **kwargs)
 
-    def restart(self, **kwargs):
+    def restart(self, **kwargs):  # noqa
```

Comment:
no test changes needed since it is net new, and also no real tests expect in the client since this is a very scheduler centric and we only really have static tests for that, which wouldn't work here.